### PR TITLE
[New Rule] Unusual Linux System Owner or User Discovery Activity

### PR DESCRIPTION
**Pull request by randomuserid**
## Issues

https://github.com/elastic/mechagodzilla/issues/122

## Summary

ML rule to accompany the unusual system owner / user discovery job

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/ml/ml_linux_system_user_discovery.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/09/03"
+
+[rule]
+anomaly_threshold = 75
+author = ["randomuserid"]
```
Comment (bm11100) [inline rules/ml/ml_linux_system_user_discovery.toml:9]:
```suggestion
author = ["Elastic"]
```
All ML rules should be updated to `Elastic` as author

File: rules/ml/ml_linux_system_user_discovery.toml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    Uncommon user command activity can be due to an engineer logging onto a server instance in order to perform manual
+    troubleshooting or reconfiguration.
+    """,
+]
+from = "now-45m"
+interval = "15m"
+license = "Elastic License"
+machine_learning_job_id = "linux_system_user_discovery"
+name = "Unusual Linux System Owner or User Discovery Activity"
+note = "note"
```
Comment (brokensound77) [inline rules/ml/ml_linux_system_user_discovery.toml:27]: TODO

**Review by brokensound77 — COMMENTED** [review]
LGTM once note is updated or removed

**Review by devonakerr — APPROVED** [review]
Straight-forward, very useful anomaly.

File: rules/ml/ml_linux_system_user_discovery.toml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+[metadata]
+creation_date = "2020/09/03"
+ecs_version = ["1.5.0"]
```
Comment (devonakerr) [inline rules/ml/ml_linux_system_user_discovery.toml:3]: Should this be bumped to 1.6.0?

Comment (devonakerr) [reply rules/ml/ml_linux_system_user_discovery.toml:3]: Otherwise looks good.

Comment (bm11100) [inline rules/ml/ml_linux_system_user_discovery.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

Comment (randomuserid) [reply rules/ml/ml_linux_system_user_discovery.toml:3]: isn't this intended to indicate the minimum required version? they were tested in 1.5 so they work in either, 1.5 is the min.

**Review by bm11100 — APPROVED** [review]

**Pull request merged**